src/accsr/remote_storage.py

```diff
@@ -3,10 +3,11 @@
 from dataclasses import dataclass, field
```

Comment:
some docstrings are not formatted correctly which is why the CI pipeline fails. You can execute a local build to make sure CI will run through, either through tox or through direct invocation of the build scripts (see README-dev.md)